handbook/product/design/documents_templates/index.md

```diff
@@ -4,9 +4,23 @@ We have several handy templates that we use to get going quick on new work.
 
 ## Figma
 
-- [Helpers & Tooling](https://www.figma.com/files/project/12944215/Helpers-%26-Tooling?fuid=872030793297791611) project includes:
-  - [Project Template](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/Project-Template), which you can duplicate and use for all new Figma files. It includes some pages we’ve found work well for supporting async work and Figma files as a source of truth.
-  - [Figma Basics for Sourcegraphers](https://www.figma.com/file/JH7GVilasQ11c9DdjpZUIF/Figma-for-Sourcegraphers-Part-1) and [Intro to designing with Wildcard DS](https://www.figma.com/file/DrlzSZOwHqIgmpHwMqCWOs/Intro-to-designing-with-Wildcard-DS), which introduce Figma and how we use it in design to anyone who may not be familiar with it.
+[Helpers & Tooling](https://www.figma.com/files/project/12944215/Helpers-%26-Tooling?fuid=872030793297791611) project includes:
+
+- [Project Template](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/Project-Template), which you can duplicate to start new Figma files. This template is pre-structured with pages that support async work and maintain Figma files as a source of truth. Designers can use the pages within this template to their discretion on a file basis:
+  - [Cover:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=0%3A1) a project thumbnail used to identify files in Figma. Use the underlying cover symbol to change the stages (draft, in progress, done, deprecated) as the project progresses.
+  - [Master:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=246%3A11) used for final, usually high-fidelity, mockups. The template showcased on this page can be used for grouping designs in bigger projects.
+  - [Prototypes:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=262%3A11) used to separate interactive prototypes that may include stages and variants that designers don't want to showcase on the main design pages. These prototypes can be actively used for usability testing, so be cautious in making changes.
+  - [Wireframes:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=103%3A3) used for the low-fidelity mockups.
+  - [Buddy review:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=260%3A11) used for reviewing the in-progress work by other team memebrs or designers. Buddy reviews are often used when a designer would like feedback from an individual(s) before sharing with a larger group.
```

Comment:
```suggestion
  - [Buddy review:](https://www.figma.com/file/JzufQnpTQtreyfnA3qpmfz/?node-id=260%3A11) used for reviewing the in-progress work by other team members or designers. Buddy reviews are often used when a designer would like feedback from an individual(s) before sharing with a larger group. Using the buddy review section of the project is helpful for simplifying review for others, because it focuses on just the elements for review.
```